.ci/install_kata_image.sh

```diff
@@ -95,6 +97,48 @@ check_not_empty() {
 	fi
 }
 
+build_image_with_dracut() {
+	image_output=${1}
+	os_version=${2}
+	agent_commit=${3}
```

Comment:
I'm always paranoid about quoting variables "just in case" :)

Reply:
Sure, good point.  I'll fix it.